diagnostic.md

```diff
@@ -33,7 +44,11 @@ y = 10;
 After each line of code executes, what are the values of x and y?  Do the variables change?  Do their values?
 
 <!-- solution below -->
+The value of x is: 3
+The value of y is: 10
```

Comment:
Fine, but please note that the questions asks for the values of x and y after _each_ line of code.